etc/docker/dev/docker-compose-localhost.yml

```diff
@@ -0,0 +1,81 @@
+version: "2"
```

Comment:
We should probably rename this to `docker-compose-storage-localhost.yml`
@tbeerman changed this in the recent PR, so it would reflect that change as well.